bad_packages.md

```diff
@@ -0,0 +1,43 @@
+A few errors found by randomly installing some lwt packages.
+
+All seem to point to libraries using lwt.unix but only linking to lwt.
```

Comment:
I agree with breaking those packages, and in this case, I think there is no technical reason not to do it (e.g. this alone won't create any inconsistent dependency diamonds). But since we still have to maintain the `META`, and warn reverse dependency maintainers about various other things, I think it's not too much extra work to do this nicely :)